t4.cpp

```diff
@@ -1,4 +1,6 @@
 namespace {
 intt;
```

Comment:
variable 'intt' is non-const and globally accessible, consider making it const [cppcoreguidelines-avoid-non-const-global-variables]